### ISSUE TITLE
Preprocessor directive recognised when not entirely preceded by horizontal whitespace

### ISSUE THREAD

**Issue by atifaziz** [atifaziz/CSharpMinifier#1]
Using version 1.0.1:

```
$ echo "foo #bar baz" | csmin tokens -f line
Text 0/1:1...3/1:4 "foo"
WhiteSpace 3/1:4...4/1:5 " "
PreprocessorDirective 4/1:5...12/1:13 "#bar baz"
NewLine 12/1:13...13/2:1 "\n"
```

**Comment by atifaziz** [comment]
This is not a bug because minifier results are undefined if input C# source is invalid in the first place. Still, it would be good to not recognise regions that are not preceded by horizontal whitespace entirely.

**Comment by atifaziz** [comment]
Closed by 1e8a1899e5dad43a52278d95db9a11f3e6c7e4f7.
